library/dokku_clone.py

```diff
@@ -73,12 +73,13 @@ def dokku_clone(data):
         data['app'],
         data['repository'])
     try:
-        subprocess.check_call(command, shell=True)
+        subprocess.check_output(command, stderr=subprocess.STDOUT, shell=True)
         is_error = False
         has_changed = True
         meta['present'] = True
     except subprocess.CalledProcessError as e:
-        meta['error'] = str(e)
+        is_error = True
+        meta['error'] = e.output
```

Comment:
Can you spotcheck the other libraries to see if they are also exhibiting this same sort of error?

Reply:
I checked about ~10 other cases of `except subprocess.CalledProcessError` and didn't see anything